src/main/flight/imu.c

```diff
@@ -332,6 +348,63 @@ STATIC_UNIT_TESTED void imuMahonyAHRSupdate(float dt, float gx, float gy, float
     q.y += (+buffer.w * gy - buffer.x * gz + buffer.z * gx);
     q.z += (+buffer.w * gz + buffer.x * gy - buffer.y * gx);
 
+#else
+    const quaternion k1 = {
```

Comment:
(this is IMO overkill. You will gain more by constructing rotation quaternion properly (cos(alpha)*[w], sin(alpha)*[x,y,z]), see iNav implementation, possibly using low-order approximation for sin/cos)

Reply:
It is overkill. I used it to see if the current integration method might have been the source of the problem I observed during the wobble tests. The integrations doesn't appear to be the source of the drift so this will be removed.